okhttp/src/main/java/okhttp3/internal/Platform.java

```diff
@@ -123,6 +126,16 @@ public void log(String message) {
     System.out.println(message);
   }
 
+  public static List<String> alpnProtocolNames(List<Protocol> protocols) {
+    List<String> names = new ArrayList<>(protocols.size());
+    for (int i = 0, size = protocols.size(); i < size; i++) {
```

Comment:
I see. Thanks for the info !

> On 05 Sep 2016, at 18:17, Jonathan <notifications@github.com> wrote:
> 
> In okhttp/src/main/java/okhttp3/internal/Platform.java:
> 
> > @@ -123,6 +126,16 @@ public void log(String message) {
> >      System.out.println(message);
> >    }
> >  
> > +  public static List<String> alpnProtocolNames(List<Protocol> protocols) {
> > +    List<String> names = new ArrayList<>(protocols.size());
> > +    for (int i = 0, size = protocols.size(); i < size; i++) {
> it is from https://speakerdeck.com/jakewharton/eliminating-code-overhead-square-hq-2015
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>